### DISCUSSION TITLE
How to link a domain to a project using the API

### DISCUSSION THREAD

**Discussion by 619** [vercel/community#3146] [Help] [answered]
### Summary

So I've noticed that you can only use the API to add domains to either the team or your personal account (https://vercel.com/guides/how-do-i-add-a-domain-using-the-vercel-api). How do I then connect the added domain to a project so that the project is accessible from the added domain (using the API)? Or is there an API call that adds the domain directly to the project?

### Steps to Reproduce

1. Call API to add domain to team/personal account.
2. It's not linked to the project it's meant to be used for, and you cannot access the project though that domain.

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out, @619. It looks like that guide describes how to add a domain to a Vercel account, but not how to link it to a specific project. Instead of that, you should use `https://api.vercel.com/v10/projects/{idOrName}/domains` to [add a domain to a project](https://vercel.com/docs/rest-api/endpoints#add-a-domain-to-a-project)
